tests/functional/fixtures/.env

```diff
@@ -1,2 +1,2 @@
 GITLAB_IMAGE=gitlab/gitlab-ee
-GITLAB_TAG=15.4.0-ee.0
+GITLAB_TAG=15.8.1-ee.0
```

Comment:
I wonder why renovate didn't pick it up

Reply:
@lmilbaum it did, see the bumps in https://github.com/python-gitlab/python-gitlab/pull/2493. I think John just wanted a clean PR for the changes needed for GitLab :)

Reply:
@lmilbaum as @nejch said it did pick it up. But updating to this new version of GitLab is breaking our tests. So need to fix the tests before we can bump it. So rather than update the PR which is updating many dependencies I created this PR with just the one dependency update.